README.md

```diff
@@ -17,25 +17,25 @@
 Add the library as a dependency to your project's build.gradle file:
 
 ```groovy
-repositories {
-	jcenter()
-	mavenCentral()
-}
-
 dependencies {
-	compile 'com.liferay.mobile:liferay-push:1.0.2'
+	compile 'com.liferay.mobile:liferay-push:1.1.0'
 }
 ```
 
 ## Use
 
 ### Registering a device
 
-To receive push notifications, your app must register itself to the portal first. On the portal side, each device is tied to a user. Each user can have multiple registered devices. A device is represented by a device token string. Google calls this the `registrationId`.
+To receive push notifications, your app must register itself to the portal first. On the portal side, each
+device is tied to a user. Each user can have multiple registered devices. A device is represented by a device token string. Google calls this the `registrationId`.
+
+To register a device, we need a `SENDER_ID`, the id of our project in firebase. Read [Firebase's documentation](https://firebase.google.com/docs/cloud-messaging/) to learn how to get the `SENDER_ID`.
 
-Read [Android's documentation](http://developer.android.com/google/gcm/client.html) to learn how to get the `registrationId`.
+The `SENDER_ID` is available, after creating a firebase project, in the *Cloud Messaging* tab under the project settings:
 
-It's easy to obtain a `registrationId` with *Liferay Push for Android*, you just have to call to the following method:
+<img src="docs/images/Firebase console Sender id.png">
+
```

Comment:
I can't see the images here https://github.com/nhpatt/liferay-push-android/blob/master/README.md